caching-service/src/main/resources/application.yml

```diff
@@ -37,6 +37,7 @@ logging:
         com.netflix.discovery.DiscoveryClient: OFF
         org.springframework.boot.web.embedded.tomcat.TomcatWebServer: INFO
         org.apache.http.conn.ssl.DefaultHostnameVerifier: DEBUG #logs only SSLException
+        org.apache.tomcat.util.net.SSLUtilBase: ERROR
```

Comment:
maybe in debug profile it could be back to the other level?

Reply:
Yes you're right, it should be debug